debug:                          false    # bool, specifies whether or not to show the unvalid TLs
# I/O Topics
input_topic:                    "/AS/P/ccat/cones"
output_full_topic:              "/AS/P/tracklimits/full"
output_partial_topic:           "/AS/P/tracklimits/partial"


# Markers
publish_markers:                true    # bool, whether or not to publish the markers
triangulation_topic:            "/AS/P/urinay/markers/triangulation"
markers_full_topic:             "/AS/P/urinay/markers/full"
markers_partial_topic:          "/AS/P/urinay/markers/partial"
